### PR TITLE
fix: replace base rpc urls with public rpcs

### PR DESCRIPTION
**Pull request by just-toby**
## Description

when we launched we didn't know of any public RPCs to use as fallbacks. now we have a [list of options](https://chainlist.org/?search=8453) and we can stop using our own API keys for fallbacks. especially quicknode since we're paying overages on our account (see link below)

_Slack thread:_ https://uniswapteam.slack.com/archives/C047U65H422/p1693418598305729

### test plan

successfully swap on BASE

### CODE REVIEW

File: src/constants/networks.ts [hunk 1]
```diff
@@ -112,10 +108,10 @@ export const FALLBACK_URLS = {
...
     // "Safe" URLs
     'https://mainnet.base.org',
-    // "Unsafe" URLs
-    QUICKNODE_BASE_RPC_URL,
-    'https://base-mainnet.blastapi.io/b5a802d8-151d-4443-90a7-699108dc4e01',
-    'https://svc.blockdaemon.com/base/mainnet/native?apiKey=zpka_1334e7c450464d06b6e33a972a7a4e57_75320f43',
+    'https://base.gateway.tenderly.co',
+    'https://base.publicnode.com',
+    'https://1rpc.io/base',
+    'https://base.meowrpc.com',
```
Comment (JFrankfurt) [inline src/constants/networks.ts:114]: image

do we want to include the 1rpc and meowrpc urls? I don't see them in the linked code

https://chainid.network/chains.json

**Review by cbachmeier — COMMENTED** [review]
LGTM once Jordan's comment is addressed

**Review by JFrankfurt — APPROVED** [review]

File: src/constants/networks.ts [hunk 2]
```diff
@@ -107,9 +107,12 @@ export const FALLBACK_URLS = {
   ],
   [ChainId.BASE]: [
     // "Safe" URLs
-    'https://mainnet.base.org',
+    'https://mainnet.base.org/',
+    'https://developer-access-mainnet.base.org/',
     'https://base.gateway.tenderly.co',
+    'wss://base.gateway.tenderly.co',
```
Comment (cbachmeier) [inline src/constants/networks.ts:113]: not sure if we can connect with websocket urls, can you confirm?

Comment (just-toby) [reply src/constants/networks.ts]: i'll just remove since it's a duplicate

**Review by cbachmeier — APPROVED** [review]

**Pull request merged**